hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java

```diff
@@ -151,7 +151,11 @@ private void initCleanerChain(String confKey) {
     this.cleanersChain = new LinkedList<>();
     String[] logCleaners = conf.getStrings(confKey);
```

Comment:
nit:  We can even rename this and related var names also.. This is not just logCleaner..  Hfile cleaner also.. may be call just cleaner or fileCleaner.

Reply:
Can do in follow-on. Just reusing what was there ....